### ISSUE TITLE
GE - No crosshair sway after using lean

### ISSUE THREAD

**Issue by EternalHabit** [emoose/xenia#18]
Basically what the title says. When zooming in the crosshair stays completely still after using the lean feature. Then goes back to normal after walking. Nothing too major but figured i'd report my findings =p

**Comment by Millennium-Eagle** [comment]
That may be an intrinsic bug, rather than an emulator issue. (Also, one I'd rather not be fixed, myself)

I've been mulling over how to mark such issues, as I have a couple that I'm not sure are emulator specific, myself. Outside of the obvious things such as missing textures, which are confirmed to be inherent bugs and not a problem with the emulator.

Perhaps marking it with an (i) or (inherent) after the game title? For example, "GE (inherent)," "PD (inherent)." If an inherent bug is important enough that a workaround is needed, I see no reason why it shouldn't be looked at.

**Comment by EternalHabit** [comment]
> 
> 
> That may be an intrinsic bug, rather than an emulator issue. (Also, one I'd rather not be fixed, myself)
> 
> I've been mulling over how to mark such issues, as I have a couple that I'm not sure are emulator specific, myself. Outside of the obvious things such as missing textures, which are confirmed to be inherent bugs and not a problem with the emulator.
> 
> Perhaps marking it with an (i) or (inherent) after the game title? For example, "GE (inherent)," "PD (inherent)." If an inherent bug is important enough that a workaround is needed, I see no reason why it shouldn't be looked at.

Yeah your right, it's a bug with the game. I just used original Xenia with a controller and it does that same thing.
